core/jazz-ui/src/app/pages/environment-codequality/env-codequality-section.component.ts

```diff
@@ -172,19 +171,15 @@ export class EnvCodequalitySectionComponent implements OnInit {
         };
       });
 
-    filterData.yMax = 1.1 * (data
-      .map((point) => {return point.y})
-      .reduce((a, b) => {
-      return Math.max(a, b);
-    }));
-    filterData.yMin = .9 * (data
-      .map((point) => {return point.y})
-      .reduce((a, b) => {
-      return Math.min(a, b);
-    }));
+    filterData.yMax = values.length ? 1.1 * (values
```

Comment:
do you think these values should be config driven?

Reply:
Yes, I think the best solution would be to use values send back from the api when you query the data. For now surya and I have agreed to use a 90-110% solution.